docs/index.rst

```diff
@@ -1,22 +1,92 @@
 .. bambi documentation master file, created by
-   sphinx-quickstart on Sun Apr  2 15:22:20 2017.
+   sphinx-quickstart on Sun Apr  6 15:22:20 2019.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
 BAyesian Model-Building Interface (BAMBI) in Python
 ===================================================
 
-API Reference:
+.. title:: Bambi Documentation
 
-.. toctree::
-   :maxdepth: 2
+.. image:: https://travis-ci.org/bambinos/bambi.svg?branch=master
+    :target: https://travis-ci.org/bambinos/bambi
+
+.. image:: https://coveralls.io/repos/github/bambinos/bambi/badge.svg?branch=master
+    :target: https://coveralls.io/github/bambinos/bambi?branch=master
+
+
+Bambi is a high-level Bayesian model-building interface written in Python. It's built on top of the `PyMC3 <https://github.com/pymc-devs/pymc3>`_ probabilistic programming framework, and is designed to make it extremely easy to fit mixed-effects models common in social sciences settings using a Bayesian approach.
```

Comment:
We should rephrase this to something like.

Bambi is a high-level Bayesian model-building interface written in Python. It works with two probabilistic programming frameworks, `PyMC3 <https://docs.pymc.io/>` or `PyStan <https://pystan.readthedocs.io/en/latest/>`, and is designed to make it extremely easy to fit Bayesian mixed-effects models common in biology, social sciences and other disciplines.

Reply:
Sure! I think I just stole that from the README.md, should I update the readme too?